src/stemmablenote.ts

```diff
@@ -60,7 +60,7 @@ export abstract class StemmableNote extends Note {
         // codeFlagDown = codeFlagUp + 1,, if not defined, code should be 0
         this.getStemDirection() === Stem.DOWN
           ? String.fromCodePoint((glyphProps.codeFlagUp?.codePointAt(0) ?? -1) + 1)
-          : glyphProps.codeFlagUp ?? '\u0000';
+          : glyphProps.codeFlagUp  || '\u0000';
```

Comment:
How come this was switched the other way? If so, please add a comment to explain the reasoning, as it is not obvious.

The comment has a small typo. Please change to:
```
// codeFlagDown = codeFlagUp + 1
// if codeFlagUp is undefined, codePoint will be 0
```

Reply:
reverted, comment updated.